.changeset/large-snakes-march.md

```diff
@@ -0,0 +1,6 @@
+---
+'firebase': major
+'@firebase/vertexai': major
+---
+
+Release VertexAI in Firebase for general availability.
```

Comment:
I don't think this is applicable anymore. Maybe you should rebase on the`main` branch, or wait for v11 to be merged before this is reviewed?

Reply:
Rebase onto main, should be good to review now